nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -639,6 +645,22 @@ in
         '';
       };
 
+      mimeTypes = mkOption {
+        type = with types; submodule {
+          freeformType = types.attrsOf (types.listOf str);
+        };
+        default = {};
```

Comment:
```suggestion
        default = null;
```
I would propose to check against null here.

Reply:
error: A definition for option `services.nginx.mimeTypes' is not of type `attribute set of list of string'.